docker-compose.yml

```diff
@@ -5,13 +5,16 @@ services:
     build: .
     depends_on:
       - api
+      - redis
     ports:
       -  "5100:5100"
   api:
-    build: https://github.com/youpin-city/youpin-api.git
+    build: ../youpin-api
```

Comment:
Should we stick to URL? So, it will get the latest image of API and cut off the local dependency.